message/infrastructure/amqp/topology_builder.go

```diff
@@ -0,0 +1,54 @@
+package amqp
+
+import (
+	"github.com/ThreeDotsLabs/watermill"
+	"github.com/pkg/errors"
+	"github.com/streadway/amqp"
+)
+
+type topologyBuilder interface {
```

Comment:
I think that it will be a good idea to make it public (may be useful for example for Dependency Injection)

Reply:
buildTopology can be also public :)